### PR TITLE
Mariocube.Uwizard version 1.1.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102595)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
Portable

File: manifests/m/Mariocube/Uwizard/1.1.1/Mariocube.Uwizard.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.Uwizard
+PackageVersion: 1.1.1
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Other%20Consoles/Wii%20U/Uwizard/Uwizard%201.1.1.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/m/Mariocube/Uwizard/1.1.1/Mariocube.Uwizard.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

**Comment by stephengillie** [comment]
image

**Comment by Trenly** [comment]
Seems the package flags don't apply retroactively.

**Pull request closed without merging**